grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -1134,7 +1134,9 @@ def section_streaming_actors(outer_panels):
                     "on average. Then we divide this duration by 1 second and show it as a percentage.",
                     [
                         panels.target(
-                            f"avg(rate({metric('stream_actor_output_buffer_blocking_duration_ns')}[$__rate_interval])) by (fragment_id, downstream_fragment_id) / 1000000000",
+                            f"sum(rate({metric('stream_actor_output_buffer_blocking_duration_ns')}[$__rate_interval])) by (fragment_id, downstream_fragment_id) \
+                            / sum({metric('stream_dispatcher_count')}) by (fragment_id, downstream_fragment_id) \
+                            / 1000000000",
```

Comment:
If we scale here, I think we may need to change `interval_ms` as well in the dashboard.

Reply:
Don't quite get this. The scale here is always in seconds. Also I guess you're referring to the query in `src/dashboard/prometheus.rs`.